src/components/InventoryTable/InventoryTable.js

```diff
@@ -349,4 +357,21 @@ InventoryTable.propTypes = {
   showNoGroupOption: PropTypes.bool, // group filter option
 };
 
-export default InventoryTable;
+const InventoryTableWrapper = forwardRef(
+  ({ needIsolatedStore = false, ...props }, ref) => {
+    const Wrapper = needIsolatedStore ? Provider : React.Fragment;
+    const store = needIsolatedStore ? useMemo(() => getStore(), []) : undefined;
```

Comment:
This is calling a hook conditionally, it should rather be this and call the hook an each render, otherwise React may throw an error about this and the change of hooks called: 
```js
const store = useMemo(() => {
  return needIsolatedStore ? getStore() undefined
}, [needIsolatedStore]);
```

Reply:
I have added it 👍🏼